packages/dapp-svelte-wallet/react-ui/src/App.js

```diff
@@ -1,22 +1,85 @@
 /* eslint-disable react/display-name */
 import React from 'react';
+import { Switch, Route } from 'react-router-dom';
+import { CssBaseline } from '@material-ui/core';
+import {
+  createTheme,
+  makeStyles,
+  ThemeProvider,
+} from '@material-ui/core/styles';
 
-import WalletConnection from './components/WalletConnection';
+import AppBar from './components/AppBar';
+import NavMenu from './components/NavMenu';
+import Contacts from './views/Contacts.js';
+import Dapps from './views/Dapps.js';
+import Dashboard from './views/Dashboard.js';
+import Purses from './views/Purses.js';
+import Issuers from './views/Issuers.js';
 
-import './App.css';
-import { withApplicationContext } from './contexts/Application';
+const appBarHeight = '64px';
+const navMenuWidth = '240px';
 
-const App = ({ connectionState }) => {
+const appTheme = createTheme({
+  palette: {
+    background: {
+      default: '#ffffff',
+    },
+  },
+  typography: {
+    fontFamily: ['"Roboto"', '"Helvetica"', '"Arial"', 'sans-serif'].join(','),
```

Comment:
Using literal double-quotes here seems inconsistent with other usage.  I could see that `'"Roboto","Helvetica","Arial",sans-serif'` is a legitimate style value, but it seems strange to add the quotes if they are not needed.